File: structure/single/protcol/c#/caller/center_call_servercaller.cs
/*this caller file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.IO;

namespace caller
{
    public class center_call_server : juggle.Icaller 
    {
        public center_call_server(juggle.Ichannel _ch) : base(_ch)
        {
            module_name = "center_call_server";
        }

        public void reg_server_sucess()
        {
            ArrayList _argv = new ArrayList();
            call_module_method("reg_server_sucess", _argv);
        }

        public void close_server()
        {
            ArrayList _argv = new ArrayList();
            call_module_method("close_server", _argv);
        }

    }
}

File: protcol/js/caller/center_call_servercaller.js
/*this caller file is codegen by juggle for js*/
function center_call_server_caller(ch){
    Icaller.call(this, "center_call_server", ch);

    this.reg_server_sucess = function(){
        var _argv = [];
        this.call_module_method.call(this, "reg_server_sucess", _argv);
    }

    this.close_server = function(){
        var _argv = [];
        this.call_module_method.call(this, "close_server", _argv);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Icaller.prototype;
    center_call_server_caller.prototype = new Super();
})();
center_call_server_caller.prototype.constructor = center_call_server_caller;

